src/tap/test/disallow_one_site_gaps2.tcl

```diff
@@ -0,0 +1,15 @@
+source "helpers.tcl"
+read_lef Nangate45/Nangate45_tech.lef
+read_lef Nangate45/Nangate45_stdcell.lef
+read_def disallow_one_site_gaps.def
+
+set def_file [make_result_file disallow_one_site_gaps2.def]
+
+place_endcaps -endcap_vertical "TAPCELL_X1"
+place_tapcells -distance 20 \
+    -master "TAPCELL_X1" \
+    -disallow_one_site_gaps
+
+write_def $def_file
+
+diff_file disallow_one_site_gaps.defok $def_file
```

Comment:
Why does this use the defok from another test?

Reply:
since these two tests need to generate the same result, so I just used this file again, if you prefer to split defok, I can do that